services/conference/src/scripts/components/map/ParticipantsLayer/Participant.tsx

```diff
@@ -78,7 +78,7 @@ const RawParticipant: React.ForwardRefRenderFunction<HTMLDivElement , Participan
             <div className={classes.pointerRotate}>
               <Pointer className={classes.pointer} />
             </div>
-            <div className={[classes.avatar, transform.counterRotationClass].join(' ')}>
+            <div className={[classes.avatar, transform.counterRotationClass, 'handle'].join(' ')}>
```

Comment:
Longer class name like 'draggableHandle' might be better for observing its relationship with <DraggableCore>.

Reply:
@zlguo1996 updated the css class name